rbac/management/group/definer.py

```diff
@@ -57,6 +57,24 @@ def seed_group(tenant):
                 logger.info("Finished seeding default group %s for tenant %s.", name, tenant.schema_name)
             else:
                 logger.info("Default group %s is managed by tenant %s.", name, tenant.schema_name)
+
+            # Default admin group
+            admin_name = "Default admin access"
+            admin_group_description = (
+                "This group contains the roles that all admin users inherit by default. "
+                "Adding or removing roles in this group will affect permissions for all admin users in your org."
```

Comment:
Same here?
```suggestion
                "Adding or removing roles in this group will affect permissions for all org admin users in your org."
```

Reply:
I changed it so now it specifies org admin users in the group description.